pages/linux/xterm.md

```diff
@@ -0,0 +1,23 @@
+# xterm
+
+> The xterm command is a terminal emulator for the X Window System.
+
+- Open the terminal with a title of `Example`:
+
+`xterm -T {{Example}}`
+
+- Open the terminal in fullscreen mode:
+
+`xterm -fullscreen`
+
+- Open the terminal with a dark blue background and yellow font:
+
+`xterm -bg {{darkblue}} -fg {{yellow}}`
+
+- Open the terminal in screen position x=200,y=20 with 100 columns and 35 rows:
+
+`xterm -geometry {{100}}x{{35}}+{{200}}+{{20}}`
+
+- Open the terminal using a monospaced font and a font-size equal to 20:
```

Comment:
Isn't monospaced font the default? If so, maybe it would be more useful to show a different example (e.g. a serif, proportional font).

Reply:
Also, I believe "font size" doesn't need a hyphen 🤓

Reply:
@xRuiAlves don't forget to change the description as well ;)